mmv1/products/vmwareengine/NetworkPeering.yaml

```diff
@@ -0,0 +1,177 @@
+    path: "response"
+  status: !ruby/object:Api::OpAsync::Status
+    path: "done"
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: "error"
+    message: "message"
+
+import_format: ["projects/{{project}}/locations/global/networkPeerings/{{name}}"]
+autogen_async: true
+
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: "vmware_engine_network_peering_ven"
+    primary_resource_id: "vmw-engine-network-peering"
+    vars:
+      name: "sample-network-peering"
+    test_env_vars:
+      region: :REGION
+  - !ruby/object:Provider::Terraform::Examples
+    name: "vmware_engine_network_peering_standard"
+    primary_resource_id: "vmw-engine-network-peering"
+    vars:
+      name: "sample-network-peering"
+    test_env_vars:
+      region: :REGION
+
+parameters:
+  - !ruby/object:Api::Type::String
+    name: "name"
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The ID of the Network Peering.
+
+properties:
+  - !ruby/object:Api::Type::Time
+    name: 'createTime'
+    output: true
+    description: |
+      Creation time of this resource.
+      A timestamp in RFC3339 UTC "Zulu" format, with nanosecond resolution and
+      up to nine fractional digits. Examples: "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z".
+
+  - !ruby/object:Api::Type::Time
+    name: 'updateTime'
+    output: true
+    description: |
+      Last updated time of this resource.
+      A timestamp in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine
+      fractional digits. Examples: "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z".
+
+  - !ruby/object:Api::Type::String
+    name: 'peerNetwork'
+    required: true
+    description: |
+      The relative resource name of the network to peer with a standard VMware Engine network.
+      The provided network can be a consumer VPC network or another standard VMware Engine network.
+
+  - !ruby/object:Api::Type::Boolean
+    name: exportCustomRoutes
+    default_value: true
+    send_empty_value: true
+    description: |
+      True if custom routes are exported to the peered network; false otherwise.
+
+  - !ruby/object:Api::Type::Boolean
+    name: importCustomRoutes
+    default_value: true
+    send_empty_value: true
+    description: |
+      True if custom routes are imported from the peered network; false otherwise.
+
+  - !ruby/object:Api::Type::Boolean
+    name: exportCustomRoutesWithPublicIp
+    default_value: true
+    send_empty_value: true
+    description: |
+      True if all subnet routes with a public IP address range are exported; false otherwise.
+
+  - !ruby/object:Api::Type::Boolean
+    name: importCustomRoutesWithPublicIp
+    default_value: true
+    send_empty_value: true
+    description: |
+      True if custom routes are imported from the peered network; false otherwise.
+
+  - !ruby/object:Api::Type::Enum
```

Comment:
Since this is a server output property. Lets just make this string type. You can link to the state values in the description.

Markdown works
 https://cloud.google.com/vmware-engine/docs/reference/rest/v1/projects.locations.global.networkPeerings#State

Reply:
Done